components/org.wso2.carbon.identity.remotefetch.core/src/main/java/org/wso2/carbon/identity/remotefetch/core/executers/tasks/RemoteFetchConfigurationBatchTask.java

```diff
@@ -0,0 +1,317 @@
+
+        RepositoryManagerComponent repositoryManagerComponent =
+                getRepositoryManagerComponentFromRegistry(remoteFetchConfiguration);
+        if (repositoryManagerComponent != null) {
+
+            return buildRepositoryManagerFromComponent(repositoryManagerComponent, remoteFetchConfiguration);
+
+        } else {
+            throw new RemoteFetchCoreException("RepositoryManager " + remoteFetchConfiguration
+                    .getRepositoryManagerType()
+                    + " is not registered in RemoteFetchComponentRegistry");
+        }
+    }
+
+    /**
+     * This method is used to retrieve RepositoryManagerComponent from registry for particular remoteFetchConfiguration.
+     * @param remoteFetchConfiguration
+     * @return
+     */
+    private RepositoryManagerComponent getRepositoryManagerComponentFromRegistry
+                (RemoteFetchConfiguration remoteFetchConfiguration) {
+
+        return this.componentRegistry
+                .getRepositoryManagerComponent(remoteFetchConfiguration.getRepositoryManagerType());
+    }
+
+    /**
+     * This method is used to build repository manager from RepositoryManagerBuilder.
+     * @param repositoryManagerComponent
+     * @param remoteFetchConfiguration
+     * @return
+     * @throws RemoteFetchCoreException
+     */
+    private RepositoryManager buildRepositoryManagerFromComponent(RepositoryManagerComponent repositoryManagerComponent,
+                                                                  RemoteFetchConfiguration remoteFetchConfiguration)
+            throws RemoteFetchCoreException {
+
+        try {
+            RepositoryManagerBuilder repositoryManagerBuilder = repositoryManagerComponent
+                    .getRepositoryManagerBuilder();
+
+            return repositoryManagerBuilder.addRemoteFetchConfig(remoteFetchConfiguration)
+                    .addRemoteFetchCoreConfig(RemoteFetchServiceComponentHolder.getInstance()
+                            .getFetchCoreConfiguration())
+                    .build();
+
+        } catch (RepositoryManagerBuilderException e) {
+            throw new RemoteFetchCoreException("Unable to build " + remoteFetchConfiguration
+                    .getRepositoryManagerType()
+                    + " RepositoryManager", e);
+        }
+    }
+
+    /**
+     * This method is used to build ConfigDeployer for particular remote fetch configuration.
+     * @param remoteFetchConfiguration
+     * @return
+     * @throws RemoteFetchCoreException
+     */
+    private ConfigDeployer buildConfigDeployer(RemoteFetchConfiguration remoteFetchConfiguration)
+            throws RemoteFetchCoreException {
+
+        // Get an instance of ConfigDeployer from registry.
+        ConfigDeployerComponent configDeployerComponent =
+                getConfigDeployerComponentFromRegistry(remoteFetchConfiguration);
+
+        if (configDeployerComponent != null) {
+
```

Comment:
Unnecessary new lines. Fix all the instances.

Reply:
Thank you, This comment and your previous comments were fixed in
https://github.com/wso2-extensions/identity-fetch-remote/pull/40
https://github.com/wso2-extensions/identity-fetch-remote/pull/41
https://github.com/wso2-extensions/identity-fetch-remote/pull/42
https://github.com/wso2-extensions/identity-fetch-remote/pull/43
https://github.com/wso2-extensions/identity-fetch-remote/pull/44
https://github.com/wso2-extensions/identity-fetch-remote/pull/45.
Hence I resolve this conversation.